app/api/v1/responses/common_response.py

```
@@ -1,24 +1,18 @@
-            api_key=api_key.api_key,
-            permissions=api_key.permissions,
-            last_time_checked=api_key.last_time_checked,
-            game_account_uuid=api_key.game_account_uuid
-        )
+    def map_response(cls, token_info: dict) -> Self:
+        try:
+            return TokenInfoResponse(
+                api_key=token_info["id"],
+                permissions=token_info.get("permissions", []))
+        except KeyError as ke:
+            raise ValueError(f"Missing required key in token_info: {ke.args[0]}") from k
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)